app/src/main/java/space/taran/arknavigator/mvp/model/repo/index/ResourceMeta.kt

```diff
@@ -20,31 +20,28 @@ data class ResourceMeta(
 
     companion object {
 
-        fun fromPath(
-            path: Path,
-            indexFailedPathCallback: IndexFailedPathCallback
-        ): ResourceMeta? {
+        fun fromPath(path: Path): Result<ResourceMeta?> {
             val size = Files.size(path)
             if (size < 1) {
-                return null
+                return Result.success(null)
```

Comment:
`null` doesn't look like success

Reply:
Although we were just ignoring it before. But I think we can emit warning that empty file is detected.